pkg/api/dirs_test.go

```diff
@@ -305,16 +305,16 @@ func TestDirs(t *testing.T) {
 				)
 			}
 
-			validateIsPermanentRedirect := func(t *testing.T, fromPath, toPath string) {
```

Comment:
So before disabling features, you should understand how and where they are used. The presence of this test should be clear that this is a feature.